tests/roles/mariadb_copy/tasks/ospdo_mariadb_copy.yaml

```diff
@@ -0,0 +1,144 @@
+
+- name: set OSPdO env vars
+  block:
+  - name: register source_mariadb_ip
+    no_log: "{{ use_no_log }}"
+    delegate_to: "{{ adoption_ospdo_host }}"
+    ansible.builtin.shell: |
+      {{ controller1_ssh }} sudo cat /var/lib/config-data/puppet-generated/haproxy/etc/haproxy/haproxy.cfg | grep -A1 'listen mysql'|tail -n 1|sed 's/.*bind //g' |sed 's/:3306.*//g'
+    register: source_mariadb_ip
+
+  - name: register source_db_root_pass
+    no_log: "{{ use_no_log }}"
+    delegate_to: "{{ adoption_ospdo_host }}"
+    ansible.builtin.shell: |
+      oc get secret -n openstack tripleo-passwords -o jsonpath='{.data.*}'| base64 -d |grep MysqlRootPassword|sed 's/.*: //g'
+    register: source_db_root_pass
+
+  - name: set src MariaDB copy shell vars
+    no_log: "{{ use_no_log }}"
+    ansible.builtin.set_fact:
+      mariadb_copy_shell_vars_src: |
+        MARIADB_IMAGE=quay.io/podified-antelope-centos9/openstack-mariadb:current-podified
+        STORAGE_CLASS=host-nfs-storageclass
+        SOURCE_MARIADB_IP={{ source_mariadb_ip.stdout }}
+        declare -A SOURCE_GALERA_MEMBERS
+        SOURCE_GALERA_MEMBERS=(
+        ["standalone.localdomain"]={{ source_mariadb_ip.stdout }}
+        )
+        SOURCE_DB_ROOT_PASSWORD={{ source_db_root_pass.stdout }}
+
+  - name: start an adoption mariadb helper pod on ospdo src env
```

Comment:
this pod is on the ospdo env , startred there also to get access to mariadb DBs, because the local maria_helper can't reach the mariadb ip inside the ospdo env